components/BaseFileUpload.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <div class="BaseFileUpload">
-    <dropzone
-      id="dropzone"
-      ref="el"
-      :awss3="awss3"
-      :options="options"
-      :destroy-dropzone="true"
-      :duplicate-check="true"
-      @vdropzone-complete="complete"
-      @vdropzone-s3-upload-error="s3UploadError"
-      @vdropzone-s3-upload-success="s3UploadSuccess"/>
+    <div v-show="showDropzone">
```

Comment:
If a different preview image should be shown, then the dropzone and its native preview feature should be hidden.

I used `v-show` instead of `v-if` because the element needs to be in the DOM, and `v-show` will just use css to hide it. There's a ref in the mounted hook below that needs it.